src/Errors.js

```diff
@@ -48,6 +48,9 @@ class Errors {
      */
     get(field) {
         if (this.errors[field]) {
+            if(typeof this.errors[field] === 'string' || this.errors[field] instanceof String) {
```

Comment:
Thanks for that example. We'll only need to do the `typeof` check as the package will never receive a string object from the server. Could you make that change?